app/views/admin/group_types/_group_type_card.html.erb

```diff
@@ -0,0 +1,24 @@
+<div class='card'>
+  <div class='card-header'>
+    <h5 class='mb-0'>
+      <button class='btn btn-link' data-toggle='collapse' data-target='#<%= group_type.name.parameterize(separator: '_') %>' aria-expanded='true' aria-controls='collapseOne'>
+        <%= group_type.name.pluralize %>
+      </button>
+      <%= link_to [:admin, group_type], method: :delete, data: { confirm: I18n.t('group_types.confirm') }, class: 'float-right' do %>
+        <%= icon('trash') %>
+      <% end %>
+      <%= link_to [:edit, :admin, group_type], class: 'float-right' do %>
+        <%= icon('pencil') %>
+      <% end %>
+      <%= link_to [:new, :admin, group_type, :user_group], class: 'float-right' do %>
+        <%= icon('plus') %>
+      <% end %>
+    </h5>
+  </div>
+
+  <div id='<%= group_type.name.parameterize(separator: '_') %>' class='collapse' aria-labelledby='headingOne' data-parent='#accordion'>
```

Comment:
It started out expanded but difficult to see the groups so I've kept it collapsed. A better option might be to have a menu option for each group type rather than them all on one index page.
But I did fix the aria-expanded and some other accessibility issues I spotted.